docs/netdata-cloud/authentication-and-authorization/api-tokens.md

```diff
@@ -2,9 +2,7 @@
 
 ## Overview
 
-Every single user can get access to the Netdata resource programmatically. It is done through the API Token which
-can be also called as Bearer Token. This token is used for authentication and authorization, it can be issued
-in the Netdata UI under the user Settings:
+Every single user can get access to the Netdata resource programmatically. It is done through the API Token which can be also called as Bearer Token. This token is used for authentication and authorization, it can be issued in the Netdata UI under the user Settings:
```

Comment:
```suggestion
Every single user can get access to the Netdata resource programmatically. It is done through the API Token, also called Bearer Token. This token is used for authentication and authorization, it can be issued in the Netdata UI under the user Settings:
```